src/main/java/blackjack/view/OutputView.java

```diff
@@ -38,8 +37,18 @@ public static void printPlayerCard(Player player) {
         System.out.println(player.getName() + " 카드: " + generateCardsMessage(player.getCards()));
     }
 
+    private static String generateCardsMessage(List<Card> cards) {
+        return cards.stream()
+                .map(OutputView::generateCardMessage)
+                .collect(joining(", "));
+    }
+
+    private static String generateCardMessage(Card card) {
+        return card.getDenomination().getName() + card.getSuit().getName();
+    }
```

Comment:
getter를 이렇게 쓰는게 아니라 card에게 DenominationName이 뭔지 알게 한다면 더 좋지 않을까 합니다 :)
```suggestion
    private static String generateCardMessage(Card card) {
        return card.getDenomination() + card.getSuit();
    }
```
이렇게 되어야 하지 않을까 합니다 :)

Reply:
수정하여 반영하였습니다!

```java
private static String generateCardMessage(Card card) {
    return card.getDenomination() + card.getSuit();
}
```